src/core/openthread-core-default-config.h

```diff
@@ -416,6 +416,16 @@
 #endif  // OPENTHREAD_CONFIG_STORE_FRAME_COUNTER_AHEAD
 
 /**
+  * @def OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE
+  *
+  * Define to 1 if you want use legacy transmit done.
+  *
+  */
+#ifndef OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE
```

Comment:
This configuration is intended to be very temporary (all other platform owners have agreed to update their drivers).  Do we need to include this here?